packages/common/src/hooks/useHostContext.ts

```diff
@@ -6,13 +6,15 @@ import { SupportedLocales } from '../intl/intlHelpers';
 
 type HostContext = {
   appBarButtonsRef: React.MutableRefObject<null>;
+  appBarExtraRef: React.MutableRefObject<null>;
   locale: SupportedLocales;
   setLocale: (locale: SupportedLocales) => void;
 };
 
 export const useHostContext = create<HostContext>(set => ({
   locale: LocalStorage.getItem('/localisation/locale') ?? 'en',
   appBarButtonsRef: createRef(),
+  appBarExtraRef: createRef(),
```

Comment:
I couldn't think of a name 😭  

I also wondered if instead of just a ref for "all this area below" it should be more specific.. lhs/rhs/center kind of thing

Reply:
`appBarContentRef` ?
`appBarBodyRef` - more like an HTML document or `<Table />`
:shrug:

at this stage no need for a left/right/centre that I know of, so I'm thinking to avoid the flex hell that will give us and just use a big box for now. could be convinced though..